src/antsibull_docs/data/sphinx_init/build_sh.j2

```diff
@@ -4,12 +4,26 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 # Created with antsibull-docs @{ antsibull_docs_version }@
+# cspell: disable cprv
```

Comment:
We don't use cspell in this repository (whether or not we should is a separate discussion), so it'd be better not to have comments for it cluttering up the sources.